### PR TITLE
Replaces cables with redstone

### PR DESCRIPTION
**Pull request by Kmc2000**
## About The Pull Request

Hi /tg/! It's me again

I've seen a lot of buzz about "smart cables" recently and well, when I looked at them I knew exactly what needed to be done.

Several players have complained that they're too simple to use, and I think they're on to something. So in return, I've made them thematically much more in-line with how they feel to use in game. Working with redstone in ss13 makes a lot more sense because of how easy it is to place these cables, and how they end up linking.

Demo video:
https://streamable.com/lbtz2

Engineering has also been renamed:
Chief Engineer -> Chief Redstone Engineer
Station Engineer -> Redstone Engineer

I do hope that you'll consider this pull request and give constructive feedback so that we can make truly epic redstone piston houses on /tg/.

## Why It's Good For The Game

The way cables link feels inconsistent and out of place. This replaces them with something I'm more used to so that the mechanic of autocompleting cables makes more sense.

## Changelog
:cl: Kmc2000
tweak: Cables have been replaced by redstone
/:cl:

### CODE REVIEW

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
Make them bright red and glowing when charged

**Comment by nemvar** [comment]
:-1: Didn't change ``MAXCOIL`` to 64

**Comment by dr-bee** [comment]
Keep this for april fools. this is gold!

**Comment by wesoda25** [comment]
God bless

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -364,15 +364,15 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list (new/datum/stack_recipe("cable restrai
...
 	throwforce = 0
 	w_class = WEIGHT_CLASS_SMALL
 	throw_speed = 3
 	throw_range = 5
 	materials = list(MAT_METAL=10, MAT_GLASS=5)
 	flags_1 = CONDUCT_1
 	slot_flags = ITEM_SLOT_BELT
-	attack_verb = list("whipped", "lashed", "disciplined", "flogged")
+	attack_verb = list("engineered", "explained complicated engineering principles to", "powered", "slapped")
 	singular_name = "cable piece"
```
Comment (nemvar) [inline code/modules/power/cable.dm]: :eyes:

**Comment by GalacticSwan** [comment]
finally, some good fucking updates

**Comment by ishitbyabullet** [comment]
unironically make it a subtype of cable.

**Comment by mrhugo13** [comment]
if oranges doesn't merge this then orange man bad eternal

**Comment by ishitbyabullet** [comment]
he doesn't have to merge the meme of renames and complete cable replacement.

but redstone being a type of cable that we could spawn would be funny as fuck.

**Comment by mrhugo13** [comment]
also
>April  Fools
Is there not a Late April Fools tag?
Smh what are you /TG/ people doing over there?

File: code/modules/power/cable.dm (hunk 1 above)
Comment (81Denton) [reply code/modules/power/cable.dm]: lol

**Comment by Dorsisdwarf** [comment]
This is the best PR

**Comment by zxaber** [comment]
At least the four-way crosses will look decent.

**Comment by CRITAWAKETS** [comment]
This should be able to be made post-april 1st via a recipe involving bronze dust, gold dust and plasma.

**Comment by PolithunIvan** [comment]
Okay this is epic

File: code/modules/power/cable.dm (hunk 1 above)
Comment (ShizCalev) [inline code/modules/power/cable.dm]: fix this and i'll merge it.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/power/cable.dm (hunk 1 above)
Comment (Kmc2000) [reply code/modules/power/cable.dm]: fix how?

**Comment by 81Denton** [comment]
We're not worthy

**Pull request closed without merging**